library/public/utils.h

```diff
@@ -68,6 +69,26 @@ class F3D_EXPORT utils
   [[nodiscard]] static std::filesystem::path collapsePath(
     const std::filesystem::path& path, const std::filesystem::path& baseDirectory = {});
 
+  /**
+   * Get an environnement variable value, returns std::nullopt if not set
+   */
+  [[nodiscard]] static std::optional<std::string> getEnv(const std::string& env);
+
+  /**
+   * Enumeration of supported Windows known folders
+   */
+  enum class KnownFolder : unsigned char
+  {
+    APPDATA,
```

Comment:
Let's use the official name
```suggestion
    ROAMINGAPPDATA,
```